test/component/deceased/testAlias.js

```diff
@@ -1,16 +1,19 @@
 const TestWrapper = require('test/util/TestWrapper'),
+    {set} = require('lodash'),
     DeceasedOtherNames = require('app/steps/ui/deceased/otherNames/index'),
     DeceasedMarried = require('app/steps/ui/deceased/married/index'),
     DeceasedDod = require('app/steps/ui/deceased/dod/index');
 
+
 describe('deceased-alias', () => {
-    let testWrapper;
+    let testWrapper, sessionData;
```

Comment:
We don't need to declare `sessionData` here as it can be scoped to the test that uses it.